src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -1102,8 +1105,11 @@ void Replayer<I>::handle_copy_image(int r) {
 
   {
     std::unique_lock locker{m_lock};
+    m_last_snapshot_bytes = m_snapshot_bytes;
     m_bytes_per_snapshot(m_snapshot_bytes);
     auto time = ceph_clock_now() - m_snapshot_replay_start;
```

Comment:
It's better to make the type explicit here:
```suggestion
    utime_t duration = ceph_clock_now() - m_snapshot_replay_start;
```